Projects/2-Intermediate/Madlib-Central.md

```
@@ -0,0 +1,43 @@
+# Madlib Central
+
+**Tier:** 1 - Beginner 
+
+The MadLib is a classic way for programmers to begin their coding journey! However, with this project, you'll be taking this one step further with an app that stores a library of Madlibs that can be contributed to as well as used to create stories. In case you don't know what a MadLib is, it's essentially a story with a few blanks in it that will be filled in by the user, except the user is unaware of what the contents are of the story they are contributing to. For example. "Once, a (noun) went (present-progressive verb)." Hopefully, your stories will be much more creative than that.
+
+On that note, get creative with your UI (User Interface)! How will you make a library not feel ovewhelming? How will you make actually going through and inputting the words to an existing Madlib feel easy? These are all things to consider while you're programming. 
+
+### Constraints
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)